esignet-core/src/main/java/io/mosip/esignet/core/spi/ClientManagementService.java

```diff
@@ -37,4 +36,21 @@ public interface ClientManagementService {
      */
     ClientDetail getClientDetails(String clientId) throws EsignetException;
 
+    /**
+     * API to register relying party client
+     * @param clientDetailCreateV2Request
+     * @return
+     * @throws EsignetException
+     */
+    ClientDetailResponse createOIDCClientV2(ClientDetailCreateV2Request clientDetailCreateV2Request) throws EsignetException;
+
+    /**
+     * API to update registered relying party client
```

Comment:
in this javadoc can we also add info about support for client name language map

Reply:
please check this comment [5a0b048](https://github.com/mosip/esignet/pull/265/commits/5a0b04816d36a5791956c438e0181a232c3f231a)